packages/experiments/src/components/Stack/StackItem/StackItem.test.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { mount } from 'enzyme';
+import { Stack } from '../index';
+
+describe('Stack Item', () => {
+  it('allows className from child component to be rendered', () => {
+    const wrapper = mount(
+      <Stack>
+        <div className="test" />
+      </Stack>
+    );
+
+    expect(wrapper.find('div.test').length).toBe(1);
+  });
+
+  it('can handle not having a class in a child of an explicit Stack.Item component', () => {
```

Comment:
Should this say 'can handle having a class...' instead of 'can handle not having a class...'?